admin/packaging.py

```diff
@@ -606,6 +606,11 @@ def __str__(self):
         # E.g.
         # /opt/flocker/lib/python2.7/site-packages/sphinx/locale/bn/LC_MESSAGES/sphinx.mo
         'file-not-in-%lang',
+
+        # Twisted 16.6 includes an executable C source file.
```

Comment:
What do we use this for? Im not familiar with iocsupport.c so that would help in the comments for other readers.

Assuming we build this before and changing this to use a binary?